mlrun/api/crud/model_monitoring/grafana.py

```diff
@@ -126,6 +128,8 @@ async def grafana_list_endpoints(
 
     table = mlrun.common.schemas.GrafanaTable(columns=columns)
     for endpoint in endpoint_list.endpoints:
+        if filter_router and endpoint.status.endpoint_type == 2:
+            continue
```

Comment:
magic number? what does it mean "2"?

Reply:
"2" represents the router model endpoint as defined in the enum class `EndpointType`:
```
class EndpointType(enum.IntEnum):
    NODE_EP = 1  # end point that is not a child of a router
    ROUTER = 2  # endpoint that is router
    LEAF_EP = 3  # end point that is a child of a router
```

 I should have used the enum class of course instead of that value... will fix it :)